### ISSUE TITLE
Needs cursors and icons, too

### ISSUE THREAD

**Issue by HubKing** [heqro/colorschemeswapper-plasmoid#8]
The GUI currently has only the colour set settings. But a desktop is more than just the colour set. The icon set and the cursor set also need to change when switching between light/dark themes. I guess, I could find some sort of terminal commands to do so and add them here, but that's cumbersome, and probably most users don't know how to do that.

Why not also support cursors and icons? I had used a similar extension for Gnome desktop and it had cursor/icon settings.

![image](https://github.com/user-attachments/assets/570f891d-b625-4bfe-9e24-9ecfb6c444e6)

![image](https://github.com/user-attachments/assets/5595fa65-c528-4d42-8b44-6ad83d72efa4)

**Comment by Goldaracena** [comment]
> Why not also support cursors and icons?

[In a previous issue](https://github.com/heqro/colorschemeswapper-plasmoid/issues/7), it was seen that changing (for instance) the Konsole profile entailed some usage of the terminal, as you needed to know the name of the icon package in advance. It was seen that the available themes could not be readily listed unless manual search was carried out. The same thing happens, for instance, [when changing the icon theme](https://zren.github.io/2020/04/28/how-to-change-plasma-icon-theme-in-the-terminal). You have to use `plasma-changeicons` and know full well the actual name of the icon theme you want to apply. For this reason, I settled for adding a textbox for executing any commands the user may want to.

Long story short: I do not want to manually search any user/system folders for icon themes or any other goodies. In fact, this widget is a frontend for `plasma-apply-colorscheme`, a KDE built-in program for changing the color scheme - it wouldn't exist if this program did not feature the option to list existing color schemes.
